.github/workflows/ci.yml

```diff
@@ -4,19 +4,18 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.20.1, 1.21.1]
+        go-version: [1.20.x, 1.21.x, 1.22.x, 1.23.0-rc.2]
```

Comment:
Maybe time to update these to reflect the state of the current versions.

Reply:
update in aab5b006e856a51306d607c70fa473b073040b66